contentctl/input/director.py

```diff
@@ -68,17 +67,19 @@ def addContentToDictMappings(self, content: SecurityContentObject):
             # Since SSA detections may have the same name as ESCU detection,
             # for this function we prepend 'SSA ' to the name.
             content_name = f"SSA {content_name}"
-        if content_name in self.name_to_content_map:
+        
+        if content_name in self.name_to_content_map and isinstance(self.name_to_content_map[content_name], type(content)):
```

Comment:
I am not sure why this change was made.  This check should enforce that no two pieces of content should have the same name (and the check below enforces that no two pieces of content should have the same uuid).  
This modification breaks that because it allows two **different content types** to have the same name, which they should not.

Reply:
I think I understand now why this change was made - there was a naming confllict in security_content between the osquery MACRO and the osquery DATASOURCE. This has been resolved by renaming the osquery macro to osquery_macro.
Unique names are a key component of contentctl (and how we map from string->content), so this MUST be maintained.